### PR TITLE
Add coolhigh canbus code and its config file

### PR DESCRIPTION
**Pull request by wfgsy101**
add coolhigh canbus code and add /apollo/modules/calibration/data/ch

### CODE REVIEW

File: modules/calibration/data/ch/calibration_table.pb.txt [hunk 1]
```diff
@@ -0,0 +1,7920 @@
+control_period: 0.01
```
Comment (Capri2014) [inline modules/calibration/data/ch/calibration_table.pb.txt]: This file should not be submitted here.

File: modules/calibration/data/ch/camera_params/front_camera_intrinsics.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#CAR 7
+header:
```
Comment (Capri2014) [inline modules/calibration/data/ch/camera_params/front_camera_intrinsics.yaml]: This one also.

File: modules/calibration/data/ch/camera_params/long_camera_extrinsics.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+header:
+  seq: 0
+  stamp:
+    secs: 0
```
Comment (Capri2014) [inline modules/calibration/data/ch/camera_params/long_camera_extrinsics.yaml]: This one also should not be here.

File: modules/calibration/data/ch/camera_params/long_camera_intrinsics.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+header: 
+  seq: 0
+  stamp: 
+    secs: 0
+    nsecs:         0
```
Comment (Capri2014) [inline modules/calibration/data/ch/camera_params/long_camera_intrinsics.yaml]: This one also should not be here.

File: modules/calibration/data/ch/camera_params/short_camera_extrinsics.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+header:
+  seq: 0
+  stamp:
+    secs: 0
```
Comment (Capri2014) [inline modules/calibration/data/ch/camera_params/short_camera_extrinsics.yaml]: This one also should not be here.

File: modules/calibration/data/ch/camera_params/short_camera_intrinsics.yaml [hunk 6]
```diff
@@ -0,0 +1,21 @@
+header: 
+  seq: 0
+  stamp: 
+    secs: 0
+    nsecs:         0
+  frame_id: short_camera
+height: 1080
```
Comment (Capri2014) [inline modules/calibration/data/ch/camera_params/short_camera_intrinsics.yaml]: This one also should not be here.

File: modules/calibration/data/ch/camera_params/start_leopard.launch [hunk 7]
```diff
@@ -0,0 +1,67 @@
+<launch>
+  <arg name="traffic_short_topic_name" default="/apollo/sensor/camera/traffic/image_short" />
+  <arg name="traffic_long_topic_name" default="/apollo/sensor/camera/traffic/image_long" />
+  <arg name="front_obstacle_6mm_topic_name" default="/apollo/sensor/camera/obstacle/front_6mm" />
+  <arg name="frame_rate" default="30"/>
+
+  <group ns="camera_long">
+    <node pkg="nodelet" type="nodelet" name="camera_nodelet_manager" args="manager" respawn="true" />
+
```
Comment (Capri2014) [inline modules/calibration/data/ch/camera_params/start_leopard.launch]: This one also should not be here.

File: modules/calibration/data/ch/camera_params/start_obstacle_camera.launch [hunk 8]
```diff
@@ -0,0 +1,26 @@
+<launch>
+  <arg name="front_obstacle_6mm_topic_name" default="/apollo/sensor/camera/obstacle/front_6mm" />
+  <arg name="frame_rate" default="30"/>
+
+
+  <group ns="camera_obstacle">
+    <node pkg="nodelet" type="nodelet" name="camera_nodelet_manager" args="manager" respawn="true" />
+
+    <node pkg="nodelet" type="nodelet" name="sensor_camera_obstacle" output="screen"
+          args="load usb_cam/UsbCamNodelet camera_nodelet_manager" respawn="true" >
```
Comment (Capri2014) [inline modules/calibration/data/ch/camera_params/start_obstacle_camera.launch]: This one also should not be here.

File: modules/calibration/data/ch/gnss_params/ant_imu_leverarm.yaml [hunk 9]
```diff
@@ -0,0 +1,21 @@
+#carid:mkz058
+#set primary if single-antenna;set both primary and secondary if dual-antenna.
+leverarm:
+  primary:
+    offset:
```
Comment (Capri2014) [inline modules/calibration/data/ch/gnss_params/ant_imu_leverarm.yaml]: This one also should not be here.

File: modules/calibration/data/ch/gnss_params/gnss_conf.pb.txt [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+        device: "/dev/ttyACM0"
+        baud_rate: 460800
+    }
+}
+
+rtk_from {
+    format: RTCM_V3
+    ntrip {
+        address: "111.111.111.11"
+        port: 0000
```
Comment (Capri2014) [inline modules/calibration/data/ch/gnss_params/gnss_conf.pb.txt]: This one also should not be here.

File: modules/calibration/data/ch/init_params/long_camera_extrinsics.yaml_init [hunk 11]
```diff
@@ -0,0 +1,17 @@
+header:
+  seq: 0
+  stamp:
+    secs: 0
+    nsecs: 0
+  frame_id: short_camera
+child_frame_id: long_camera
+transform:
```
Comment (Capri2014) [inline modules/calibration/data/ch/init_params/long_camera_extrinsics.yaml_init]: This one also should not be here.

File: modules/calibration/data/ch/init_params/radar_extrinsics.yaml_init [hunk 12]
```diff
@@ -0,0 +1,17 @@
+header:
+  seq: 0
+  stamp:
+    nsecs: 0
+    secs: 0
+  frame_id: short_camera
+child_frame_id: radar
```
Comment (Capri2014) [inline modules/calibration/data/ch/init_params/radar_extrinsics.yaml_init]: This one also should not be here.

File: modules/calibration/data/ch/init_params/short_camera_extrinsics.yaml_init [hunk 13]
```diff
@@ -0,0 +1,17 @@
+header:
+  seq: 0
+  stamp:
+    secs: 0
+    nsecs: 0
+  frame_id: velodyne64
+child_frame_id: short_camera
```
Comment (Capri2014) [inline modules/calibration/data/ch/init_params/short_camera_extrinsics.yaml_init]: This one also should not be here.

File: modules/calibration/data/ch/navigation_ch.pb.txt [hunk 14]
```diff
@@ -0,0 +1,6615 @@
...
+minimum_speed_protection: 0.1
+lat_controller_conf {
+  ts: 0.01
+  preview_window: 0
+  cf: 155494.663
+  cr: 155494.663
+  mass_fl: 520
+  mass_fr: 520
+  mass_rl: 520
+  mass_rr: 520
```
Comment (Capri2014) [inline modules/calibration/data/ch/navigation_ch.pb.txt]: This one also should not be here.

**Comment by Capri2014** [comment]
Please sign the CLA and remove those configs/launch files first.

File: modules/canbus/vehicle/ch/ch_vehicle_factory.h [hunk 15]
```diff
@@ -0,0 +1,65 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (Capri2014) [inline modules/canbus/vehicle/ch/ch_vehicle_factory.h]: 2019

File: modules/canbus/vehicle/ch/protocol/BUILD [hunk 16]
```diff
@@ -0,0 +1,214 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "canbus_ch_protocol",
```
Comment (Capri2014) [inline modules/canbus/vehicle/ch/protocol/BUILD]: Please separate them in the BUILD file, one library for each.

Comment (wfgsy101) [reply modules/canbus/vehicle/ch/protocol/BUILD]: BUILD file has been modified as you required, please help to review again, thank you

Comment (Capri2014) [reply modules/canbus/vehicle/ch/protocol/BUILD]: @wfgsy101 Did you commit the latest one? It still looks like they are put into one binary

Comment (wfgsy101) [reply modules/canbus/vehicle/ch/protocol/BUILD]: @Capri2014 the latest BUILD as below, is there still anything wrong,

cc77474535032bf50aa8788d1466b82a

Comment (Capri2014) [reply modules/canbus/vehicle/ch/protocol/BUILD]: `canbus_ch_protocol` should be depending on the binaries then.

Comment (Capri2014) [reply modules/canbus/vehicle/ch/protocol/BUILD]: Take this one as a reference: https://github.com/ApolloAuto/apollo/blob/812d3e396d48dbc3b50b7eab9af26f8cffabb546/modules/canbus/vehicle/transit/protocol/BUILD#L8

Comment (Capri2014) [reply modules/canbus/vehicle/ch/protocol/BUILD]: Sorry I mean *libraries.

Comment (wfgsy101) [reply modules/canbus/vehicle/ch/protocol/BUILD]: @Capri2014  update the BUILD，please review,  thanks

**Pull request merged**